main.py

```diff
@@ -1,45 +1,46 @@
 """
-You need to create a github repo (public), plan project structure, use MIT license. 
-Also you need to decide how to run two different conversions with a single script, e.g. how to decide which conversion you have to perform.
+import argparse - For parsing arguments when run script
 
+function def cel_to_fahren(tempreture_in_celsium): - Take temperature in Celsius and convert to Fahrenheit
 From Celsius  to Fahrenheit
 Temp: 20 C
 20 * (9/5) + 32 = 68 F
 
 
+function def fahren_to_cel(temprutere_in_fahren): - Take temperature in Fahrenheit and convert to Celsius
 From Fahrenheit to Celsius
 Temp: 68 F
 (5/9) * (68 - 32) = 20 C
 
 """
-import argparse  # For parsing arguments when run script
+import argparse
 
 
-def cel_to_fahren(tempreture_in_celsium):  # Take temperature in Celsius and convert to Fahrenheit
+def cel_to_fahren(tempreture_in_celsium):
     fahren = (tempreture_in_celsium * (9 / 5)) + 32
     return fahren
 
 
-def fahren_to_cel(temprutere_in_fahren):  # Take temperature in Fahrenheit and convert to Celsius
+def fahren_to_cel(temprutere_in_fahren):
     cels = (5 / 9) * (temprutere_in_fahren - 32)
     return cels
 
 
 parser = argparse.ArgumentParser()
 parser.add_argument("-v", "--verbose", help="It just return your argument")
-parser.add_argument("-f", help="Take temperature in Fahrenheit and convert to Celsius")
-parser.add_argument("-c", help="Take temperature in Celsius and convert to Fahrenheit")
+parser.add_argument("-f", "--fahrenheit", help="Take temperature in Fahrenheit and convert to Celsius")
+parser.add_argument("-c", "--celsius", help="Take temperature in Celsius and convert to Fahrenheit")
 
 args = parser.parse_args()
 
 if args.verbose:
     print(str(args) + " It's verbose argument")
-elif args.f:
-    # print(f"{args.f} Fahrenheit")
-    print(fahren_to_cel(int(args.f)))
-elif args.c:
-    # print(f"{args.c} Celsius")
-    print(cel_to_fahren(int(args.c)))
+elif args.fahrenheit:
+    # print(f"{args.fahrenheit} Fahrenheit")
+    print(fahren_to_cel(float(args.fahrenheit)))
```

Comment:
@MariaMaximova, you can add `type=float` to `add_argument` method and remove this conversion.